.yarn/versions/ed70b723.yml

```diff
@@ -0,0 +1,30 @@
+releases:
+  "@yarnpkg/cli": prerelease
+  "@yarnpkg/core": prerelease
+  "@yarnpkg/plugin-constraints": prerelease
+  "@yarnpkg/plugin-essentials": prerelease
+  "@yarnpkg/plugin-git": prerelease
+  "@yarnpkg/plugin-init": prerelease
+  "@yarnpkg/plugin-node-modules": prerelease
+  "@yarnpkg/plugin-npm": prerelease
+  "@yarnpkg/plugin-npm-cli": prerelease
+  "@yarnpkg/plugin-pnp": prerelease
+  "@yarnpkg/plugin-version": prerelease
```

Comment:
We don't use prerelease now that we're in stable, so it would be patch - although in this case it probably wouldn't even require a release for anything, since it only affects the types.

Reply:
The core should be released so third-party plugins can get the benefit of this as well

Reply:
Oh right, I forgot API consumers - minor, then?

Reply:
Yup

Reply:
Yeah I kept those at prerelease because I wanted to have this discussion but I ended up way too sleepy after opening the PR to add a comment.
Minor idd seems like the right choice